pallets/protos/src/lib.rs

```diff
@@ -519,15 +547,16 @@ pub mod pallet {
 			// store by owner
 			<ProtosByOwner<T>>::append(owner, proto_hash);
 
-			// index immutable data for IPFS discovery
-			transaction_index::index(extrinsic_index, data_len as u32, proto_hash);
-
-			let cid = [&CID_PREFIX[..], &proto_hash[..]].concat();
-			let cid = cid.to_base58();
-			let cid = [&b"z"[..], cid.as_bytes()].concat();
```

Comment:
So we don't create a special CID anymore for IPFS?

Reply:
Was just UX, gas expensive UX, has no purpose. Easy to derive locally. 😂